rails/etsy_product_catalog/db/migrate/20150301225625_create_categories.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCategories < ActiveRecord::Migration
+	def change
+		create_table :categories do |t|
+			t.integer :etsy_category_id
+			t.string :name
+			t.string :category_name
+			t.string :short_name
+			t.string :long_name
+			t.integer :num_children
+
+			t.timestamps null: false
+		end
+	end
+end
```

Comment:
Okay, I will edit this. I think rubymine has some defaults set up for continuous indentation that I need to change.